oadp/bookbag/workshop/content/Overview.adoc

```diff
@@ -59,7 +59,7 @@ There are mainly two categories of Velero plugins that can be specified while Ve
 - `defaultVeleroPlugins`:
 
 There are five types of default Velero plugins can be installed: 
-`AWS`, `GCP`, `Azure` and `OpenShift`, and `CSI`. For installation, 
+`AWS`, `GCP`, `Azure`,`OpenShift`, `KubeVirt` and `CSI`. For installation, 
```

Comment:
There will also be `vsm` plugin for oadp-1.2.. are we just doing 1.1 for now?

Reply:
this is just for 1.1